Python/ast_opt.c

```diff
@@ -142,15 +142,6 @@ check_complexity(PyObject *obj, Py_ssize_t limit)
         }
         return limit;
     }
-    else if (PyFrozenSet_Check(obj)) {
```

Comment:
`check_complexity` is only used in `safe_multiply` so, I deleted code which related to sets.